src/projects/Experimental.tsx

```diff
@@ -0,0 +1,92 @@
+import SwatchWithForm from '../SwatchWithForm';
+import { StitchPattern, Color } from '../types'
+import { useSwatchConfigStateFromURLParams, useEffectToUpdateURLParamsFromSwatchConfig } from '../URLSwatchParams';
+import { useState } from "react";
+import ColorSequenceInfo from '../ColorSequenceInfo'
+import DropdownInput from '../inputs/Dropdown';
+import { totalColorSequenceLength } from '../colorSequenceHelpers'
+
+const red = "#ff001d" as Color;
+const cream = "#fcf7eb" as Color;
+const ltblue = "#8dd0f2" as Color;
+const navy = "#0e0e66" as Color;
+
+type StaggerType = 'colorStretched' | 'colorSwallowed'
+
+function Experimental() {
+  const defaultSwatchConfig = {
+    colorSequence: [
+      {color: navy, length: 3},
+      {color: red, length: 3},
+      {color: navy, length: 3},
+      {color: ltblue, length: 2},
+      {color: cream, length: 5},
+      {color: ltblue, length: 2},
+    ],
+    stitchesPerRow: 18, //Note: explicitly ok not saving zero from search params here
+    numberOfRows: 40, //Note: explicitly ok not pulling zero from search params here
+    colorShift: 0,
+    staggerLengths: false,
+    stitchPattern: StitchPattern.moss,
+  }
+
+  const { swatchConfig, setSwatchConfig, setSearchParams} = useSwatchConfigStateFromURLParams(defaultSwatchConfig);
+
+  const [staggerType, setStaggerType] = useState('normal' as StaggerType)
```

Comment:
Related to my other comment, why the F does this work when StaggerType can only use 'colorStretched' | 'colorSwallowed', I don't think I understand typescript.